draft-ietf-quic-transport.md

```diff
@@ -4331,11 +4331,10 @@ Each endpoint unilaterally decides if the spin bit is enabled or disabled for a
 connection. Implementations MUST allow administrators of clients and servers
 to disable the spin bit either globally or on a per-connection basis. Even when
 the spin bit is not disabled by the administrator, implementations MUST disable
-the spin bit for a given connection with a certain likelihood. The random
-selection process SHOULD be designed such that on average the spin bit is
-disabled for at least one eighth of network paths. The selection process
-performed at the beginning of the connection SHOULD be applied for all paths
-used by the connection.
+the spin bit for at least a sixteenth of connections with an expectation that
+the spin bit is disabled for at least one eighth of network paths. The selection
+process performed at the beginning of the connection SHOULD be applied for all
+paths used by the connection.
```

Comment:
In reviewing this last sentence, I am reminded that this provides a nice signal that might improve linkability.  I think that the requirement to perform the determination separately for every network path/connection ID as suggested above is better.